src/main/java/net/mcreator/workspace/Workspace.java

```diff
@@ -404,6 +404,8 @@ public static Workspace readFromFS(File workspaceFile, @Nullable Window ui)
 			String workspace_string = FileIO.readFileToString(workspaceFile);
 			Workspace retval;
 			try {
+				workspace_string = workspace_string.replaceAll("\"type\": \"mob\",", "\"type\": \"livingentity\",")

```

Comment:
You marked this solved two times already. This needs proper implementation with deserializer.. Don't just mark things as resolved, please

Reply:
Actually I see you added this. My question is changed to why you are still doing parsing here if you added proper JSON adapter for this?

Reply:
I surely forgot to remove it.